protocol.go

```diff
@@ -168,7 +169,7 @@ func ParseFileTime(
 	if err != nil {
 		return errors.New("unable to parse ATime component of message")
 	}
-	mTime, err := strconv.Atoi(string(parts[2][0:10]))
+	mTime, err := strconv.ParseUint(string(parts[2][0:10]), 0, 32)
```

Comment:
Potential out of bounds error due to upper bound in slice (i.e., 10), I think we should check the length of the `mTilme` field and return an appropriate error if it does not match our expectations.

Reply:
Now both are checked for length